CONTRIBUTING.md

```diff
@@ -0,0 +1,140 @@
+# OSIsoft Connector Contribution Guidelines
+
+Thank you for your interest in contributing to the project.
+
+All types of contributions are valued and encouraged, though there are a few guidelines which must be followed to make the contribution process easy and consistent. See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them.
+
+If you like the project, but don't have time to contribute, there are other easy ways to show your support and appreciation, such as:
+- Star the project
+ - Refer to this project in another project
+ - Mention or share the project at local meetups and tell your colleagues
```

Comment:
why does the indentation change?

Reply:
I think this was intentional when it was first written for the starter project to make sub-lists. @alexjhawk  is this not the case?

Reply:
@oliver-walker-hms please fix this here.